src/dbnode/network/server/tchannelthrift/node/service.go

```diff
@@ -740,19 +742,22 @@ func (s *service) fetchReadEncoded(ctx context.Context,
 		encoded, err := db.ReadEncoded(ctx, nsID, tsID,
 			opts.StartInclusive, opts.EndExclusive)
 		if err != nil {
-			elem.Err = convert.ToRPCError(err)
```

Comment:
Are we aware of a reason this was originally written to not fail the batch on any `ReadEncoded` errors? Just wondering if this will move away from some specific behavior that was originally intended

Reply:
Yeah, that's a good question. @robskillington alluded to the fact that it was modeled after what was done for fetching blocks. but it's actually not necessary in this case. I think that's also corroborated by the fact that no one is actually making used of the embedded error message. I stopped short of removing `Err` from the individual element and this code path is only touched by the FetchTagged endpoint, so if folks wanted the ability to get and collect individual errors, that'd still be a possibility.

Reply:
Got it yeah make sense then how it ended up present but not actually required. Yeah seems like both pros and cons to actually fully removing it. Some niceness in keeping it consistent and having around later if we decide to use it; but also makes current code a bit more confusing since we don't use it. I'd probably vote to either remove it or just leave a comment on the field for future awareness

Reply:
seems worth removing if nobody is using it. odd to have "multiple" error paths.

Reply:
Forgot to mention here that `Err` is a field on a proto generated object, so leaving the field in place but added a comment indicating that the field is deprecated.